Makefile

```diff
@@ -108,194 +108,268 @@ BASE_LDFLAGS = ${SHRINKFLAGS} \
 
 GO_LDFLAGS = -ldflags '${BASE_LDFLAGS} ${EXTRA_LDFLAGS}'
 
+define go-build
```

Comment:
Why not `go_build` since we have `curl_to`?

Reply:
Remaned, thanks!